modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1755,6 +1863,18 @@ EFAElement3D::getFaceNeighbor(unsigned int face_id, unsigned int neighbor_id) co
     EFAError("edge neighbor does not exist");
 }
 
+EFAElement3D *
+EFAElement3D::getEdgeNeighbor(unsigned int face_id,
+                              unsigned int edge_id,
+                              unsigned int neighbor_id) const
+{
+  if (_face_edge_neighbors[face_id][edge_id].size() &&
```

Comment:
The top line in this conditional isn't necessary. This can just be:
`if (neighbor_id < _face_edge_neighbors[face_id][edge_id].size())`